File: Unit - 1/Introduction/C implementation/maxElement.c
//Program to find max element in a array
#include <stdio.h>

int maxElement(int A[], int size);
int main() 
{
    int array[] = {3, 1, 15, 2, 9};
    int size = sizeof(array) / sizeof(array[0]);
    int maxValue = maxElement(array, size);
    printf("The largest element in the array is: %d\n", maxValue);
    return 0;
}

int maxElement(int A[], int size) 
{
    int maxVal = A[0];
    for (int i = 1; i < size; i++) 
    {
        if (A[i] > maxVal) 
        {
            maxVal = A[i];
        }
    }
    return maxVal;
}

File: Unit - 1/Introduction/Java Implementation/MaxElement.java
//Program to find max element in a array
public class MaxElement {
    private static int maxElement(int[] A) {
        int maxVal = A[0];
        for (int i = 1; i < A.length; i++) {
            if (A[i] > maxVal) {
                maxVal = A[i];
            }
        }
        return maxVal;
    }

    public static void main(String[] args) {
        int[] array = {3, 1, 15, 2, 9};
        int maxValue = maxElement(array);
        System.out.println("The largest element in the array is: " + maxValue);
    }
}